lib/index.js

```diff
@@ -159,7 +175,14 @@ function getPolyfillString(options) {
 					}
 				});
 				if (builtPolyfillString) {
-
+					return streamToString(
+						browserify(intoStream(builtPolyfillString))
```

Comment:
I should probably not have this run on every requested bundle, instead only running it on ones where polyfills are written in commonjs or ES6 modules.